### DISCUSSION TITLE
det检测训练，loss降低，hmean也降低

### DISCUSSION THREAD

**Discussion by jackieZhouQQ** [PaddlePaddle/PaddleOCR#12603] [Q&A]
请提供下述完整信息以便快速定位问题/Please provide the following information to quickly locate the problem

- 系统环境/System Environment：docker镜像 paddle:2.5.1-gpu-cuda12.0-cudnn8.9-trt8.6
- 版本号/Version：Paddle：  PaddleOCR： 问题相关组件/Related components：
- 运行指令/Command Code：python3 -m paddle.distributed.launch --log_dir=./debug/ --gpus '0,1'  tools/train.py -c configs/det/ch_PP-OCRv4/stu.yml
- 完整报错/Complete Error Message：

我们提供了AceIssueSolver来帮助你解答问题，你是否想要它来解答(请填写yes/no)?/We provide AceIssueSolver to solve issues, do you want it? (Please write yes/no):no

请尽量不要包含图片在问题中/Please try to not include the image in the issue.

**Comment by jackieZhouQQ** [comment]
训练日志文件
[train.log](https://github.com/PaddlePaddle/PaddleOCR/files/12644546/train.log)

**Comment by kawaiikujira** [comment]
我也有这种情况

**Comment by jackieZhouQQ** [comment]
> 我也有这种情况

你解决了没

**Comment by kawaiikujira** [comment]
> > 我也有这种情况
> 
> 你解决了没

还没找到原因，训练的模型在训练集上的评估也不提升，你也是在自己的数据跑的吗

**Comment by jackieZhouQQ** [comment]
> > > 我也有这种情况
> > 
> > 
> > 你解决了没
> 
> 还没找到原因，训练的模型在训练集上的评估也不提升，你也是在自己的数据跑的吗

是的，500条左右标注数据

**Comment by kawaiikujira** [comment]
> > > > 我也有这种情况
> > > 
> > > 
> > > 你解决了没
> > 
> > 
> > 还没找到原因，训练的模型在训练集上的评估也不提升，你也是在自己的数据跑的吗
> 
> 是的，500条左右标注数据

有可能是数据标注问题

**Comment by jingmingtao** [comment]
> > > 我也有这种情况
> > 
> > 
> > 你解决了没
> 
> 还没找到原因，训练的模型在训练集上的评估也不提升，你也是在自己的数据跑的吗

你解决了吗，我也遇到了，近乎崩溃阿

**Comment by shsh1400** [comment]
解决了吗，同样的问题
